### PR TITLE
Search stores

### PR DESCRIPTION
**Pull request by VojinBjelica**

### CODE REVIEW

**Review by zeljkons1992 — APPROVED** [review]

**Comment by zeljkons1992** [comment]
[Need to implement a discount calculator on home screen](https://trello.com/c/BE1l75uc/9-need-to-implement-a-discount-calculator-on-home-screen)

**Comment by zeljkons1992** [comment]
[Add a screen to display all available stores where vegaIT employees have discounts.](https://trello.com/c/fkdnGIVn/5-add-a-screen-to-display-all-available-stores-where-vegait-employees-have-discounts)

File: lib/core/ui/elements/selected_store_display.dart
Comment (koceljevac) [inline lib/core/ui/elements/selected_store_display.dart:1]: magic numbers

File: lib/features/wallet/presentation/screens/wallet_screen.dart
Comment (koceljevac) [inline lib/features/wallet/presentation/screens/wallet_screen.dart:1]: again magic numbers

File: lib/features/wallet/presentation/widgets/discount_calculator.dart [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+                          style: TextStyle(
+                            fontSize: 12,
+                            fontWeight: FontWeight.bold,
+                          ),
+                        ),
+                        Text(
+                          _discountedPrice != null
+                              ? '${_discountedPrice!.toStringAsFixed(2)} rsd'
+                              : 'N/A',
+                          style: const TextStyle(
```
Comment (koceljevac) [inline lib/features/wallet/presentation/widgets/discount_calculator.dart:111]: use text style

File: lib/features/wallet/presentation/widgets/discount_calculator.dart
Comment (koceljevac) [inline lib/features/wallet/presentation/widgets/discount_calculator.dart:1]: again magic numbers create assets with size for every element

**Review by zeljkons1992 — APPROVED** [review]

**Review by koceljevac — APPROVED** [review]

**Pull request merged**